blocks/search.py

```diff
@@ -263,10 +263,19 @@ def search(self, input_values, eol_symbol, max_length,
             first iteration are ignored. This useful when the sequence
             generator was trained on data with identical symbols for
             sequence start and sequence end.
+        as_lists : bool, optional
+            If ``True``, a list of trimmed output sequences and a list of
+            their costs are returned, instead of the default output format.
 
         Returns
         -------
-        Sequences in the beam, masks, and corresponding log-probabilities.
+        outputs : :class:`numpy.ndarray`
+            The generated sequences, time is the first axis, the number in
+            the beam is the second.
+        mask : :class:`numpy.ndarray`
+            The mask for the outputs, same layout as `outputs`
```

Comment:
The thing is, the whole output format is changed when `as_lists=True`. I would hate describing all the differences here.

@bartvm , what's the right way to document outputs when those can be delivered into two different formats?